demo/console/README.md

```
@@ -0,0 +1,11 @@
+# QuickStarts
+
+This directory includes all the sample manifests for OpenShift Console
+Quick Starts.
+
+Any `QuickStart` or `ConsoleYAMLSample` can be installed via the
+following command:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)